src/desktop/src/ui/views/wallet/Receive.js

```diff
@@ -190,10 +190,28 @@ class Receive extends React.PureComponent {
                         success={t('receive:addressCopiedExplanation')}
                     >
                         <p>
-                            {receiveAddress.split('').map((char, index) => {
-                                const scrambleChar = scramble[index] > 0 ? byteToChar(scramble[index]) : null;
-                                return <React.Fragment key={`char-${index}`}>{scrambleChar || char}</React.Fragment>;
-                            })}
+                            {receiveAddress
+                                .substring(0, 81)
+                                .split('')
+                                .map((char, index) => {
+                                    const scrambleChar = scramble[index] > 0 ? byteToChar(scramble[index]) : null;
+                                    return (
+                                        <React.Fragment key={`char-${index}`}>{scrambleChar || char}</React.Fragment>
+                                    );
+                                })}
+                            <span>
+                                {receiveAddress
+                                    .substring(81, 90)
+                                    .split('')
+                                    .map((char, index) => {
+                                        const scrambleChar = scramble[index] > 0 ? byteToChar(scramble[index]) : null;
```

Comment:
The `index` is reset, so the animation would be more correct like:
```suggestion
                                        const scrambleChar = scramble[index + 81] > 0 ? byteToChar(scramble[index + 81]) : null;
```